src/cryptojwt/jwt.py

```diff
@@ -107,7 +108,7 @@ def receiver_keys(self, recv, use):
         :param use: What the keys should be usable for
         :return: A list of keys.
         """
-        return self.key_jar.get(use, owner=recv)
+        return self.key_jar.get(use, issuer_id=recv)
```

Comment:
General comment: The rename of `issuer` to `issuer_id` will cause a lot of grief with existing code using named arguments (like my code). Please reconsider this name change.

If very important, we could add aliases and support both variants.

Reply:
I've felt the grief :-( so I know what you're talking about. I still went ahead with the change.
Aliases is a good middle ground proposal.

Reply:
The only problem is of course that we'll never get rid of `owner` that way.